src/hub.ts

```diff
@@ -31,7 +31,6 @@ export class Hub extends EventEmitter {
     protected _current: number = 0;
 
     protected _bleDevice: IBLEDevice;
-    private _rssi: number = -100;
```

Comment:
I can reintroduce RSSI in this or a followup PR. In the current state the `_rssi` was never changed from the initial.

Reply:
Oh that's definitely a bug. RSSI should totally be accessible. But if it's not working at the moment it's fine to remove it in this PR - just need to remember to fix it.

Reply:
I can follow up with the RSSI readout PR. In the previous code the branch that supposed to read the value was missing: https://github.com/nathankellenicki/node-poweredup/blob/0fbd27d41ff49b435ae7df260bd06200b13c8433/src/lpf2hub.ts#L403